mappings/com/mojang/blaze3d/platform/GLX.mapping

```diff
@@ -0,0 +1,192 @@
+CLASS net/minecraft/class_6125 com/mojang/blaze3d/platform/GLX
+	FIELD field_30544 blendFuncSeperateSupported Z
+	FIELD field_30545 gl21Supported Z
+	FIELD field_30546 shadersSupported Z
+	FIELD field_30547 vboSupported Z
+	FIELD field_30549 arrayBuffer I
+	FIELD field_30551 LOGGER Lorg/apache/logging/log4j/Logger;
+	FIELD field_30552 fboMode Lnet/minecraft/class_6125$class_6126;
+	FIELD field_30553 shadersEnabled Z
+	FIELD field_30554 arbShaderObjectsEnabled Z
+	FIELD field_30555 arbMultitextureEnabled Z
+	FIELD field_30556 arbTextureEnvCombine Z
+	FIELD field_30557 gl14Supported Z
```

Comment:
separateBlend?

Reply:
inherently misleading. its assignment is either to whether gl 1.4 in the context or separate blend supported (gl 1.4 +).

Reply:
"or separate blend supported" , so seperateBlend ?

Reply:
that's a gl 1.4 thing
separate blend functions is not the only gl 1.4 + thing
this can be used to check whether stencil wrapping, auto mipmap generation, etc are supported
also this field is public

Reply:
actually nvm, Ill just add a javadoc comment stating its purpose

Reply:
realized its private
will rename